site/en/docs/webstore/faq/index.md

```diff
@@ -129,36 +125,41 @@ You may also report fake reviews via the Developer [Support Form][cws-support].
 ### Is there a limit to the number of items I can have on the Chrome Web Store? {: #faq-gen-29 }
 
 You can upload as many items to the Chrome Web Store as you like, but by default, you are limited to
-having a total of 20 published items at any one time. This limit applies to the sum of your Chrome
-Apps and Chrome Extensions as a total—it is not 20 of each item. If you reach this limit, [you may
+having a total of 20 published items at any one time. This limit applies to the sum of your Themes and Chrome Extensions as a total—it is not 20 of each item. If you reach this limit, [you may
 request a limit increase][cws-support]. The Chrome Web Store staff will review your existing items and your
 developer account history, and if approved, you will be granted an increase. Please note that if
 your developer account has been suspended in the past, or you have had items taken down previously
 for policy violations, or your items consistently receive low quality ratings, your request may be
 denied.
 
+
+## Development
+
 ### Can I obfuscate my code? {: #faq-gen-30 }
 
 As of October 1st 2018, developers must not obfuscate code or conceal functionality of their
 extensions. This also applies to any external code or resource fetched by the extension package. For
 more information, please review our [program policies][program-policies].
 
-## Development
+The following are some examples of code obfuscation:
+
+- Base 64 encoding
+- Character encoding
 
 ### How do I write an extension? {: #faq-dev-01 }
 
-You can find all the information you need to develop extensions at [/docs/extensions/][ext].
+You can find all the information you need to develop extensions in the [Documentation for Chrome extensions developers][ext].
 
 ### How can I push an update of my code to my users? {: #faq-dev-02 }
 
-Visit [https://chrome.google.com/webstore/devconsole][110] and click on the listing you wish to
+To [Update your Chrome Web Store item][update-item], visit the [developer dashboard][dev-dashboard] and click on the listing you wish to
 update. Once you upload a new version and click **Publish**, your update will automatically be
 pushed out to users over the next few hours.
 
 ### Can I host my own extension? {: #faq-dev-03 }
 
-Yes, you may host your own extension, although you will need to take care of [packaging and
-serving][111] the [auto-update manifest][112] yourself.
+Yes, you can host your own extension. However, Windows and Mac OS 
```

Comment:
This "yes" is kind of bad considering the vast majoity of users are excepted. Can you rewrite this?

e.g., "You can host your own extensions in limited cases, such as for Linux or Chrome OS users. Consider [enterprise publishing] or developer mode."

Reply:
Sounds much better! I'll change it :)